### ISSUE TITLE
Scene Numbering Does not seem to be working

### ISSUE THREAD

**Issue by slattanzio** [SickChill/sickchill#8411]
Version: Latest
OS: QNAP Docker
Browser: Chrome
What you did: Tried to download an episode 7 of Impractical Jokers
What happened: Episode 6 of impractical jokers was downloaded
What you expected: Episode 7 was downloaded

I posted an issue on the discord but it may be better to also contact here. I have a show that has messed up their episode numbers because they put a special as an episode. Now when sickchill looks to pull an episode, it is the wrong one. For example, episode 6 is the special and then episode 7 is actually episode 6. Is there a way in sickchill to account for this? I saw something about putting in a scene number so that episode 6 is actually 7 but when I try to search, it still pulls in episode 6. 

I tried to follow instructions here: https://github.com/SickChill/SickChill/wiki/Scene-exceptions-and-numbering#what-is-scene-numbering . I would then expect that when I try to pull episode 7, it would pull 8  which is actually 7 and rename appropriately but it still seems to be pulling 7 which in our example is actually episode 6. 

Looking here: https://thexem.info/xem/show/5578 it looks like the adjustment is automatically being made but it is still pulling the wrong episode. 

Please let me know if you have any questions where I can help and thanks in advanced for any help!

**Comment by slattanzio** [comment]
Thanks @BKSteve ! Im not sure where the error in the screenshot is because the Paul Sheer episode there does say 10x7 which is correct based on the torrent and the Sideline Smack Talk special is episode 6. But I look forward to knowing what you find and any potential fixes!!

**Comment by slattanzio** [comment]
Ah I see. That would work. In the past this has happened and the episode would show up in the special section instead. You can look at season 9 that is supposed to have 26 episodes but you can search up s09e27 and it is there. It is because there was a special "Highlights from the High Seas" as well as "Twists and Turns". I'm not sure why there aren't 28 episodes due to this but it is the same issue as what is happening now where the special is pulled and then the next episode pulls the wrong episode due to incorrect numbering

**Comment by miigotu** [comment]
> I'm looking into it but I would expect the display to show the Season 10 with `Sideline Smack Talk` being in the `Series 10` display area. So that S10E06 which is searched and downloaded is actually named `Sideline Smack Talk` and not `Paul Sheer` This is what you probably expected ![image](https://user-images.githubusercontent.com/10173496/231504425-5996427c-8e8d-4867-909b-3afe73bf7e14.png)
> 
> Add: seems like the scene_numbering database isn't being populated

this one is incorrect.

**Comment by miigotu** [comment]
> Took a quick look, it seems the XEM data isn't being collected or is not able to be mapped. Further investigation required. May be related to not receiving absolute numbering from theTVDB
> 
> ![image](https://user-images.githubusercontent.com/10173496/231444522-190aede5-4346-4e73-9079-4bd58713023c.png)

this is the correct way it should look

**Comment by slattanzio** [comment]
So I do see that but due to the mismatch, the list of included episodes for 7 for example has both 7 and 8 when it should only be 8. That is causing it to pull the wrong episode. Here is what I see when clicking manual snatch for 7.
<img width="859" alt="Screen Shot 2023-04-12 at 5 04 07 PM" src="https://user-images.githubusercontent.com/4605073/231584716-0c7405a1-1d62-4277-8f21-4a890c8be587.png">


Since episodes labelled with 8 are already out, when I click on manual snatch of 8 I see the following when I should see nothing because that episode comes out on Thursday

<img width="854" alt="Screen Shot 2023-04-12 at 5 03 52 PM" src="https://user-images.githubusercontent.com/4605073/231584732-20f8e474-f20f-43d7-b547-5582abafb433.png">

**Comment by miigotu** [comment]
The problem is on the mapping in https://thexem.info/xem/show/5578 was updated on 3/22 -- see: https://thexem.info/xem/changelog/5578

It was updated likely after your search pulled some results. I would shutdown sickchill, delete the wrong file, and delete cache.db, start back up and set the wrong episode to skipped and then set it to wanted. See what it pulls as results then for me.

**Comment by miigotu** [comment]
It is possible somewhere someone changed the search from ~always~ using ep.scene_episode to using ep.episode, or it was always that way. I do see a ton of calls to finding the scene info when it should just be done when searching and when refreshing the show. We really need to work on that functionality and process, once some of my other todo items are done.

**Comment by miigotu** [comment]
Curious, do you have episode nfo or xml files enabled?

**Comment by slattanzio** [comment]
I will wait until next week to see if it pulls only the correct number. But I know that last season had this problem and it was never fixed so I don't have high hopes unless this numbering feature was added recently.

I haven't edited any settings for nfo or xml and don't know where to look but it should be default. Is there somewhere specific you want me to check? The only setting I tried is in the format tab of editing the show `search by scene numbering (uncheck to search by indexer numbering)` which did nothing
